catalan.ini

```
@@ -0,0 +1,669 @@
+PowerGrip = "AGAFA"
+ScrewAttack = "ATAC ESPIRAL"
+Boots = "BOTES"
+HiJumpBoots = "SUPERSALT"
+SpaceJump = "SALT ESPACIAL"
+SpeedBooster = "ACCELERACIÓ"
+
+[Buttons]
+Button = "Botó"
+Aim = "Apuntar"
+Aim2 = "Apuntar2"
+Up = "Amunt"
+Down = "Avall"
+Left = "Esquerra"
+Right = "Dreta"
+Start = "Start"
+Select = "Select"
+Back = "Enrere"
+LB = "LB"
+RB = "RB"
+LT = "LT"
+RT = "RT"
+RightShoulder = "R"
+LeftShoulder = "L"
+ZR = "ZR"
+ZL = "ZL"
+R1 = "R1"
+R2 = "R2"
+L1 = "L1"
+L2 = "L2"
+Cross = "Ics"
+Circle = "Cercle"
+Triangle = "Triangle"
+Square = "Quadrat"
+LStick = "Estic Esq."
+RStick = "Estic Der."
+Missile = "Míssil"
+Jump = "Salt"
+Fire = "Tret"
+Backspace = "Retrocés"
+Tab = "Tabulador"
+Enter = "Enter"
+Shift = "Shift"
+RShift = "Shift Der."
+LShift = "Shift Esq."
+Control = "Control"
+RControl = "Ctrl Der."
+LControl = "Ctrl Esq."
+NumLk = "Num Lock"
+ScrollLock = "Scroll Lock"
+AltRight = "Alt Der."
+AltLeft = "Alt Esq."
+Fuita = "ESC"
+Menu = "Menú"
+Controls = "Controls"
+Alt = "Alt"
+Pause = "Pausa"
+SpaceBar = "Espai"
+PageUp = "PageUp"
+PageDown = "PageDown"
+End = "Fi"
+Home = "Home"
+Insert = "Inserir"
+Delete = "Esborrar"
+NumPad0 = "NumPad0"
+NumPad1 = "NumPad1"
+NumPad2 = "NumPad2"
+NumPad3 = "NumPad3"
+NumPad4 = "NumPad4"
+NumPad5 = "NumPad5"
+NumPad6 = "NumPad6"
+NumPad7 = "NumPad7"
+NumPad8 = "NumPad8"
+NumPad9 = "NumPad9"
+NumPadMult = "NumPad "
+NumPadPlus = "NumPad +"
+NumPadMinus = "NumPad -"
+NumPadDot = "NumPad ."
+NumPadDivide = "NumPad /"
+XAxis = "X Axis"
+YAxis = "I Axis"
+ZAxis = "Z Axis"
+RAxis = "R Axis"
+UAxis = "O Axis"
+VAxis = "V Axis"
+POV = "POV"
+
+[Misc]
+TowerCellTemp = "TEMPERATURA DE LA CÈL·LULA D'ENERGIA"
+Monster_Text = "Presència Metroid detectada a l'àrea"
+Monster2 = "Metroid 2"
+Items_Tip = "| - Objectes Aconseguits"
+Translation = "Finalitzant Traducció"
+FusionUnlock = "Dificultat Fusion desbloquejada"
+ExtrasSeed = "Llavor"
+ExtrasMode = "Mode"
+SensitivityMode = "Mode sensitivitat"
+SensitivityMode_Tip = "Redueix la intensitat d'alguns efectes de llum."
+Widescreen = "Pantalla Panoràmica"
+Widescreen_Tip = "Canvia a pantalla panoràmica. S'aplica al canviar d'habitació en el joc"
+
+
+[ExtrasMenu]
+Extras = "Extres"
+Extras_Tip = "Afegits a Revisions Posteriors"
+Title_Extras = "Extres"
+Controller = "Comandament Utilitzat"
+Controller_Tip = "Configura quins botons de comandament apareixen en les pistes del joc"
+XBOX = "Xbox"
+Playstation = "Playstation"
+NintendoPro = "Nintendo Pro"
+SNESColored = "Colors SNES"
+MOGA = "MOGA"
+OUYA = "OUYA"
+LowHealthAlarm = "Alarma de Salut Baixa"
+LowHealthAlarm_Tip = "Activa l'avís quan la salut està baixa"
+Soft = "Baixa"
+Loud = "Alta"
+EarlyBaby = "Bebè Metroid Aviat"
+EarlyBaby_Tip = "Permet que el bebè Metroid et segueixi a qualsevol lloc"
+FusionMSRsuit = "Estil Armadura Fusion"
+FusionMSRsuit_Tip = "Utilitza els colors de l'Armadura Fusion en el mode Fusion"
+Default = "Predeterminat"
+MSR = "M:SR"
+ExtremeMonsters = "Lab. Genètic Difícil"

```

Seems like it doesn't display correctly. Let's remove the dot.
```suggestion
ExtremeMonsters = "Lab Genètic Difícil"
```